source/vmware/vmware-overview.rst

```diff
@@ -0,0 +1,380 @@
+.. _minio-vmware:
+
+=======================
+MinIO on VMware vSphere
```

Comment:
@kerneltime so this is a rough initial draft, but I'd like to get your opinion on how this reads.

In particular:

1) Does this accurately describe our relationship with VMware vSphere components
2) Does this accurately describe how MinIO works on vSphere

Thanks.

Reply:
@dvaldivia if you want to squeeze in a looksie before you go on vacation I wont stop you :)